packages/active-campaign-client/src/index.ts

```diff
@@ -1,8 +1,18 @@
 import { SQSEvent } from 'aws-lambda';
 import { sqsQueueHandler } from './handlers/sqsQueueHandler';
+import { resyncUser } from './helpers/resyncUser';
 
 export async function sqsQueue(event: {
   readonly Records: SQSEvent['Records'];
 }) {
   return await sqsQueueHandler(event);
 }
+
+export async function handler(event: {
+  readonly Records: SQSEvent['Records'];
+}) {
+  //return await sqsQueueHandler(event);
+  const cognitoId = event.Records[0].body;
+  console.log('cognitoId: ', cognitoId);
+  return await resyncUser(cognitoId);
```

Comment:
I check the event and when the sqs quque fail it populate the dlq error queue with the same event that generate and error. So the type could be the same:
```suggestion
  return await resyncUserHandler(event);
```